tm2/pkg/db/pebbledb/pebbledb.go

```diff
@@ -31,6 +32,7 @@ func NewPebbleDB(name string, dir string) (*PebbleDB, error) {
 
 func NewPebbleDBWithOpts(name string, dir string, o *pebble.Options) (*PebbleDB, error) {
 	dbPath := filepath.Join(dir, name+".db")
+	dbPath = strings.ReplaceAll(dbPath, " ", "\\ ")
 	db, err := pebble.Open(dbPath, o)
```

Comment:
don't know if this work too. I am thinking we can have the issue with others non escaped char
```suggestion
	db, err := pebble.Open(fmt.Sprintf("%q", dbPath), o)
```